core/src/main/resources/lib/hudson/rssBar.jelly

```diff
@@ -24,22 +24,33 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:x="jelly:xml" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-  <div align="right" style="margin:1em">
-    <a href="${rootURL}/legend">${%Legend}</a>
-    <span style="padding-left:1em">
-      <a href="rssAll"><img src="${imagesURL}/atom.gif" border="0" alt="Feed" height="16" width="16"/></a>
+  <style type="text/css">
+    .icon-rss {
+      border: 0;
+    }
+    .rss-bar-item {
+      padding-left: 1em;
+    }
+    #rss-bar {
+      margin:1em;
+    }
+  </style>
+  <div id="rss-bar" align="right">
```

Comment:
🐛 `align` attribute is not allowed on `div` element. Moreover, you would be mixing semantic and representation.